packages/lit-analyzer/src/analyze/data/extra-html-data.ts

```diff
@@ -58,7 +58,7 @@ const HTML_5_ATTR_TYPES: { [key: string]: string | string[] | [string[]] } = {
 	title: "string",
 	manifest: "",
 	href: "string",
-	target: ["_blank", "_parent", "_self", "_top"],
+	target: 'string|"_blank"|"_parent"|"_self"|"_top"',
```

Comment:
This just reduces to string, though I think it's nice to keep for docs. Is there some level of warning that's very low like a "did you mean?". I could see there being nice suggestions if you accidentally typed `blank` instead of `_blank` for instance.

Reply:
yeah it does, i figured its good to keep the rest of the literals around for docs/autocomplete. i think normally vscode will at least show `_blank` if you start typing `blank` (as it should show it as a suggestion). but not sure how we could hint at vscode to prioritise those